### ISSUE TITLE
Importing MPChartLib in Android Studio

### ISSUE THREAD

**Issue by nbau21** [PhilJay/MPAndroidChart#350]
I can't import the library into my project. It breaks my app and upon rebuilding, it says that "plugin with id 'android-maven' not found". I've tried the solutions [here ](https://stackoverflow.com/questions/16588064/how-do-i-add-a-library-project-to-the-android-studio) and [here](https://www.youtube.com/watch?v=1MyBO9z7ojk) but I keep getting the same error.

Using the [.jar file](https://github.com/PhilJay/MPAndroidChart/releases) works, but it doesn't have the HorizontalBarChart functionality that I want for my app. 

Any suggestions?

**Comment by triggetry** [comment]
How did you include the library into your project?
For me its just about adding ``compile 'org.quanqi:mpandroidchart:1.7.5`` to the ``dependencies`` section of the ``build.gradle`` file.

**Comment by nbau21** [comment]
Second answer in this link (the one with a lot of screenies)

https://stackoverflow.com/questions/16588064/how-do-i-add-a-library-project-to-the-android-studio

**Comment by nbau21** [comment]
I managed to get it to work. Here's what I did:

Clone MPAndroidChart.

Open in Eclipse.

Right-click on MPChartLib -> Export -> Java  -> JAR File

Check 'Export generated class files and resources' and 'Export Java source files and resources'

Finish

Download the latest release.jar of MPAndroidChart

Copy the missing files and folders from that JAR file to the new JAR file. Do not overwrite anything.

Put the new JAR file into YourProject/libs/

Synchronize on Android Studio, right click on your new JAR file and select 'Add as Library'

It should work now.
